### DISCUSSION TITLE
Can I set the background for the whole screen when the LottieFullScreen is set to false?

### DISCUSSION THREAD

**Discussion by SirMathelot** [timbru31/cordova-plugin-lottie-splashscreen#433] [Q&A] [answered]
When I set the background color it only changes the color of the splashscreen, not the whole screen. Is there a way to change the background of the whole screen when the splash is shown? Alternatively is there a way to set LottieFullScreen to true and then change the size of the animation? In the example screens I have the preferences set as:

`<preference name="LottieBackgroundColor" value="#ff0000aa" />`

![screen1](https://user-images.githubusercontent.com/62252088/168078695-bd560e40-db6d-4120-b681-dab4aacea3a1.png)
![screen2](https://user-images.githubusercontent.com/62252088/168078689-78c75bbc-b20c-409f-bad5-7a6177a6ce57.png)

**Comment by SirMathelot** [comment, marked as answer]
I solved this problem by displaying the native ionic splash screen (@ionic-native/splash-screen/ngx) at the same time as the lottie splash screen (be sure to leave LottieFullScreen set to false). That way the static native splash screen served as a background for the lottie animation splash screen and when the app loaded the lottie splash screen didn't lag.

**Reply by mohammeddanish247** [reply]
thanks. i will implement it in my project
